src/lib/annotations/Annotator.js

```diff
@@ -534,12 +540,20 @@ class Annotator extends EventEmitter {
     fetchAnnotations() {
         this.threads = {};
 
+        // Do not load any pre-existing annotations if the user does not have
+        // the correct permissions
+        if (!this.permissions.canViewAllAnnotations || !this.permissions.canViewOwnAnnotations) {
```

Comment:
Does not fetch any annotations if the user cannot view any/their own annotations